bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/handler/HomeAssistantThingHandler.java

```diff
@@ -255,7 +259,7 @@ public void componentDiscovered(HaID homeAssistantTopicID, AbstractComponent<?>
      * Callback of {@link DelayedBatchProcessing}.
      * Add all newly discovered components to the Thing and start the components.
      */
-    @SuppressWarnings("null")
+    @SuppressWarnings({ "null" })
```

Comment:
Could this be simply removed ?

Reply:
There were dozens of warnings. Resolved all those in https://github.com/openhab/openhab-addons/pull/13518/commits/d3f4d85d6ed990d5eca2d67a1428e2858600d5bd

Opted for simple `SupressWarnings` in unit tests.